packages/snowpack/src/commands/build.ts

```diff
@@ -6,8 +6,10 @@ import glob from 'glob';
 import * as colors from 'kleur/colors';
 import mkdirp from 'mkdirp';
 import path from 'path';
+import {performance} from 'perf_hooks';
```

Comment:
This has support since Node v8 and is more meant for accurate profiling than `Date.now()`.